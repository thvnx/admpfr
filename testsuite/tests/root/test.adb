with Ada.Text_IO; use Ada.Text_IO;
with Admpfr;      use Admpfr;

procedure Test is
   M, N : Mpfloat;
begin
   N.Set (Float (1.4));

   M.Cbrt (N);
   Put_Line (M'Image);

   M.Rootn (N, 3);
   Put_Line (M'Image);
end Test;
