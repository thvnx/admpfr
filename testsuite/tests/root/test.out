1.1186889357310104e+00
1.1186889357310104e+00
8.9390354017092988e-01
